scripts/apps/users/components/TextWithMentions.tsx

```diff
@@ -12,23 +12,26 @@ const mentionRegexp = /@\[([^\]]+)\]\((desk|user):([^\)]+)\)/g;
  * @name TextWithMentions
  * @description Displays a text containing mentions.
  */
-export const TextWithMentions: React.StatelessComponent<any> = ({children, ...props}) => {
-    const msg = children;
-    const n = msg.length;
+
+export interface IProps {
+    message: string;
+}
+export const TextWithMentions: React.StatelessComponent<any> = ({message, ...props}: IProps) => {
```

Comment:
> `props` isn't used anywhere, you should remove it

```
React.StatelessComponent<any> = ({message, ...props}
//                                            ^ I meant from everywhere, not only interface
```

Reply:
I was about to remove it but then I thought if anyone pass other `prop` than `message` then it would be wrong.

Reply:
The interface won't allow passing any other prop than `message`. If someone wanted to pass anything else then they would add it everywhere - to the interface and to implementation.

Reply:
Okay, so I will remove it.